cli/src/cli_client.ts

```diff
@@ -384,5 +380,14 @@ const evalCommandArgs = async (command: ResourceCommandAndArguments) => {
 
 const command = checkAndGetCommandArgs();
 
+const evalCommandWrapped = async (promise: Promise<void>) => {
+  try {
+    await promise;
+  } catch (error) {
+    logMessage('Found error during processing', (error as Error).message);
```

Comment:
```suggestion
    logMessage('Error:', (error as Error).message);
```

Should be more readable since shorter